src/tools/genVersionHeader.pl

```diff
@@ -108,6 +116,7 @@
         $opt_V = $result;
         $vcs = 'Bazaar';
     }
+    $cv = `bzr version-info -q -timestamp`;  # this is untested
```

Comment:
`-timestamp` isn't valid, at least for the "bzr" which ships with Debian 11 (`Breezy (brz) 3.1.0`).

```
bzr version-info -q --custom --template='{date}'
```

prints something like `2012-01-02 10:57:46 +0100`.